arm-analysisservices/2016-05-16/swagger/analysisservices.json

```diff
@@ -525,6 +528,28 @@
           "description": "An array of administrator user identities."
         }
       }
+    },
+    "BackupConfiguration": {
+      "description": "An object that represents backup configurations",
+      "type": "object",
+      "properties": {
+        "storageAccount": {
+          "type": "string",
+          "description": "The name of storage account for backup configuration. It is in the format of 'Resource Group Name/Storage Account Name'"
```

Comment:
This is how our ARM request/response is working for database backup. For instance, for PATCH operation, the payload is like (copied from armguiclient):

{"properties":{"asAdministrators":{"members":["{admin1}","{admin2}"]},"backupConfiguration":{"storageAccount":"{sa}","blobContainer":"bc","accessKey":"ak"}},"tags":{"{key}":"{value}","{key1}":"{value1}"}}

The implementation will validate storage account, blob container and access key. I think this API change is inline with our ARM operations.